en-US/firefox/features/password-manager.lang

```diff
@@ -30,6 +28,7 @@ Password hero
 Forget the reset. Firefox Password Manager keeps all your passwords so you can log in automatically, or find saved passwords easily. For super security, give your computer a <a href="%(url)s">master password</a>.
 
 
+# Ninja is a skilled fighter trained in Japanese martial arts. Here it means a protector. Feel free to substitue with character known to local culture.
```

Comment:
I don't think it means protector here? I think ninja here means "master", "expert" or similar